client/src/store/new-request-module.ts

```diff
@@ -2019,7 +2019,7 @@ export class NewRequestModule extends VuexModule {
       return data
     } catch (error) {
       console.error('addRequestActionComment() =', error) // eslint-disable-line no-console
-      return data
+      throw new Error(`addRequestActionComment() = ${error}`)
```

Comment:
Don't do this. It will cause a chain of errors going back to the `@Action`.

Return null if you must, and handle that instead.

Reply:
ok, will try that.